### PR TITLE
Add isort to our linting toolkit

### PR DESCRIPTION
**Pull request by velovix**
This adds `isort` to our CI pipeline, which checks the order and format of imports. Developers can use `isort` to automatically do import sorting for them, and a section on this is now in the README. We configure `isort` to use the "black" profile, which should prevent any disagreements on import formatting between the two tools. However, in CI, we always run `black` last to give it the final say on formatting.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -25,9 +25,15 @@ packaging = "^20.4"
...
 black = "^19.10b0"
 mypy = "*"
+isort = "^5.6"
 
 [tool.poetry.scripts]
 brainframe = "brainframe.cli.main:main"
 
 [tool.black]
 line-length = 79
+
```
Comment (BryceBeagle) [inline pyproject.toml:35]: Glad you found that isort can be configured in the `pyproject.toml` now :+1:

**Review by BryceBeagle — CHANGES_REQUESTED** [review]
You may want to re-run isort after merging in the other PR :wink:

File: pyproject.toml [hunk 2]
```diff
@@ -25,9 +25,15 @@ packaging = "^20.4"
...
+[tool.isort]
```
Comment (BryceBeagle) [inline pyproject.toml:36]: Glad you found that isort can be configured in the `pyproject.toml` now :+1:

Comment (velovix) [reply pyproject.toml:36]: Yeah! I was very happy to see that.

**Comment by velovix** [comment]
Oh, yeah, good call! It should be ready to merge now, in that respect.

**Review by BryceBeagle — APPROVED** [review]

**Review by BestDriverCN — APPROVED** [review]

**Pull request merged**